.changeset/sixty-walls-pull.md

```diff
@@ -0,0 +1,5 @@
+---
+'@primer/react-brand': patch
+---
+
+Update `useProvidedRefOrCreate` to support functional refs
```

Comment:
As this hook isn't exposed to consumers, do you think it would be more relevant to say something like


```suggestion
Updated `Accordion`, `Checkbox`, `Radio`, `SubNav`, `Tooltip` and `VideoPlayer` components to support functional refs.
```

wdyt?

(I've omitted `Card`, `RiverAccordion` as, although they use the hook, they don't actually interact with the ref and instead just forward it on, so they've always supported functional refs. The hook could safely be removed from those two components, but I think that's outside the scope of this PR.)